storage/mysql/storage.sql

```diff
@@ -111,6 +111,10 @@ CREATE TABLE IF NOT EXISTS SequencedLeafData(
   FOREIGN KEY(TreeId, LeafIdentityHash) REFERENCES LeafData(TreeId, LeafIdentityHash) ON DELETE CASCADE
 );
 
+
+CREATE INDEX SequencedLeafMerkleIdx
+  ON SequencedLeafData(TreeId, MerkleLeafHash);
```

Comment:
Why not put this in the `CREATE TABLE` definition?

Reply:
That would involve me knowing more SQL syntax than I do...

Reply:
I'm going to leave as-is for consistency with other indices in there.